core/src/main/resources/error/error-classes.json

```diff
@@ -210,6 +210,12 @@
       "<message>"
     ]
   },
+  "INTERVAL_DIVIDED_BY_ZERO" : {
+    "message" : [
+      "Division by zero."
```

Comment:
Doesn't `try_divide` help here? The `TryDivide` expression has some examples with intervals:
https://github.com/apache/spark/blob/46175d1362062035fb93f87f25d61a9b711359ab/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/TryEval.scala#L187-L190

Reply:
Make sense. I have updated the message. Thanks!